packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListing.tsx

```diff
@@ -77,12 +83,17 @@ export const AlertListing = () => {
     CloudPulseServiceType
   >[] => {
     return serviceOptions && serviceOptions.data.length > 0
-      ? serviceOptions.data.map((service) => ({
-          label: service.label,
-          value: service.service_type,
-        }))
+      ? serviceOptions.data
+          .filter(
+            (service) =>
+              aclpServices?.[service.service_type]?.alerts?.enabled ?? false
```

Comment:
```suggestion
              aclpServices?.[service.service_type]?.alerts?.enabled
```

no need to have fallback, undefined will also be treated as false

Reply:
Having false is more clear and explicit. Same for all the below comments. In one of the linode prs, i received comment to rather explicitly define than having like above.